doc/source/Interface/ControlsAndShortcuts/index.md

```diff
@@ -0,0 +1,171 @@
+# Controls and Shortcuts #
+
+The following is a list of input device controls and shortcuts for manipulating Gaffer's _Node Graph_, _Viewer_, and _Script Editor_ tabs.
+
+
+## _Node Graph_ Tab ##
```

Comment:
This "_Italic_ Normal" combo looks odd to me. Is that standard? Again, lets call it Pane or Panel (or no suffix at all?)

Reply:
The plan is: italic = words you see on UI elements. This is fairly standard. The reason it's mixed is to distinguish UI and general concepts. _Node Graph_ tab = the actual interface or visual display of the node graph. Node graph = the graph itself, outside of any specific context. Phrases like "navigating/editing/traversing the node graph" can be extremely confusing because it doesn't specify the context. This will become very critical when I talk about the node graph conceptually.

I thought "panel" referred to the rectangular regions that hold the various views? I'm using "tab" because that's how each of these things behaves (left-to-right, select/open/close like in a web browser), and it conveys more meaning than pane. I'm guessing this will conflict with the code?

Reply:
Actually no, they are technically all (or mostly) Editors inside a TabbedContainer I suppose. But tabs are used in a lot of places (inside the UI of a NodeEditor for example) so referring to the outer editor/inspector/etc as a tab is confusing (to me).

Reply:
I found tab confusing the first time I saw it too. But the panel is really the thing holding all the tabs (see the Remove Panel menu item in the layout menu). I would refer to them as editors.

Reply:
Since it's in the code, I can stick to "editors," just as long as we're OK with referring to it as the "_Node Editor_ editor" in the text. I don't want to be inconsistent and refer to everything as the "[type] editor" but not give the Node Editor the same treatment.